include/godot_cpp/classes/wrapped.hpp

```diff
@@ -55,6 +55,7 @@ class Wrapped {
 	virtual const GDExtensionInstanceBindingCallbacks *_get_bindings_callbacks() const = 0;
 
 	void _notification(int p_what) {}
+	void _notification2(int p_what, bool p_reversed) {}
```

Comment:
I don't think we need to add `_notification2()`. In godot-cpp, we _can_ break compatibility, and we want to have the same API (as much as is reasonable) as in a Godot module. So, in my opinion, we should just change `_notification()` to have the new second argument.